packages/blue-migration-sdk-viem/src/positions/supply/compoundV3.supply.ts

```diff
@@ -0,0 +1,176 @@
+import {
+  type Address,
+  type ChainId,
+  MathLib,
+  getChainAddresses,
+} from "@morpho-org/blue-sdk";
+import { Time } from "@morpho-org/morpho-ts";
+
+import type {
+  Action,
+  SignatureRequirement,
+} from "@morpho-org/bundler-sdk-viem";
+import BundlerAction from "@morpho-org/bundler-sdk-viem/src/BundlerAction.js";
+import { baseBundlerAbi } from "@morpho-org/bundler-sdk-viem/src/abis.js";
+import {
+  type Account,
+  type Client,
+  encodeFunctionData,
+  maxUint256,
+  verifyTypedData,
+} from "viem";
+import { signTypedData } from "viem/actions";
+import { cometExtAbi } from "../../abis/compoundV3.abis.js";
+import type {
+  MigrationBundle,
+  MigrationTransactionRequirement,
+} from "../../types/actions.js";
+import {
+  MigratableProtocol,
+  SupplyMigrationLimiter,
+} from "../../types/index.js";
+import { getCompoundV3ManagerApprovalMessage } from "../helpers/signatures.js";
+import {
+  MigratableSupplyPosition,
+  type MigratableSupplyPositionConfig,
+} from "./index.js";
+
+interface MigratableSupplyPositionConfig_CompoundV3
+  extends Omit<MigratableSupplyPositionConfig, "protocol"> {
+  nonce: bigint;
+  cometAddress: Address;
+  cometName: string;
+}
+
+export class MigratableSupplyPosition_CompoundV3
```

Comment:
```suggestion
export class MigratableCompoundV3SupplyPosition
```

Genuinely curious, is there a benefit of suffixing it instead of making it part of the name?

Reply:
Easier to navigate/search + autocompletion is smoother + it makes it clear that it is an extension of `MigratableSupplyPosition`